nimbus/src/client/http_client.rs

```diff
@@ -181,7 +181,9 @@ mod tests {
             {{
                 "schemaVersion": "{current_version}.0.0",
                 "slug": "mobile-a-a-example",
-                "application": "reference-browser",
+                "application": "",
+                "appName": "reference-browser",
+                "channel": "nightly",
```

Comment:
application is being deprecated, since it is currently still required in the schema I worried that it would break to be absent.  Let me play around with making it an Option and removing it from the examples.

Reply:
Maybe use the rust #[deprecated] stuff here too?

Reply:
Turns out, serde was nifty enough I could remove `application` everywhere.